components/operator-service/src/main/java/com/wso2telco/dep/operatorservice/model/BlacklistWhitelistSearchResponseDTO.java

```diff
@@ -0,0 +1,23 @@
+package com.wso2telco.dep.operatorservice.model;
+
+public class BlacklistWhitelistSearchResponseDTO {
+
+    private boolean exists;
+    private String error;
```

Comment:
Not sure whether you need this error string here. Since the DTO is used only for blacklist search and at the same time boolean false will denote that the result is not found this variable can be removed. And if you remove the error variable you do not need the entire DTO for one boolean variable. Use the boolean straightly in all the places